features/user_can_comment_on_an_article.feature

```diff
@@ -0,0 +1,37 @@
+Feature: User can comment on an article
+    As an Author,
+    In order to provide more value to my readers,
+    I would like to be able to attach an image to the article.
+
+    Background:
+        Background:
+        Given the date is "2018-08-18"
+        Given the following article exist
+        | title          | body                                                             | 
+        | This is so sad | A recent report suggest that news are mostly sad. Which is sad.  |
+        | This is so fun | A recent report suggest that news are mostly fun. Which is fun.  |
+
+        And the following user exists
+        | email             | role       |
+        | pablo@test.com    | basic_user |
+        | pablo2@test.com   | subscriber |
+
+        Scenario: Pablo can NOT see article page
+            When I am logged in as 'pablo@test.com'
+            And I am on the 'landing' page
+            And I click on "This is so sad"
+            Then I should see 'Access denied'
+            Then I should be redirected to the 'landing' page
```

Comment:
Why are we testing for this in the context of adding comments?

Reply:
Maybe not. I was thinking that since only subscriber should be able to comment, we might want to test what happens if you're a basic_user.

Reply:
Isn’t that tested in another story?

Reply:
I guess not. Let’s go with this